module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -139,9 +139,11 @@ protected function translateList($transPrefix, $list)
     {
         $transList = [];
         foreach ($list as $current) {
-            $transList[] = $this->translate(
-                $transPrefix . $current, [], $current
-            );
+            $default = $current;
+            if (!($current instanceof \VuFind\I18n\TranslatableString)) {
```

Comment:
Can we simplify the code changes in this file by using translateWithPrefix now that we have defined that method? Should we move the TranslatableString awareness into that function as well?

Reply:
Indeed we can. :) Done.